samples/cpp/text_generation/benchmark_genai.cpp

```diff
@@ -36,10 +38,29 @@ int main(int argc, char* argv[]) try {
     size_t num_warmup = result["num_warmup"].as<size_t>();
     size_t num_iter = result["num_iter"].as<size_t>();
 
+    if (result.count("prompt_file")) {
+        prompt = utils::read_prompt(result["prompt_file"].as<std::string>());
```

Comment:
Make it an error if `prompt_file` and `prompt` are given at the same time

Reply:
Done